Source/RunbookEntrypoint.ps1

```diff
@@ -62,11 +65,11 @@ New-Item -Path $localRootPath -ItemType "directory" | Out-Null
 $allFiles = @()
 Download-AllFiles -remoteRootUri $remoteRootUri -localRootPath $localRootPath -allFiles ([ref]$allFiles)
 
-$scriptsToExecute = ($allFiles | ? { $_.Execute -eq $true })
+$scriptsToExecute = ($allFiles | Where-Object { $_.Execute -eq $true })
 $scriptNum = 0
 foreach ($scriptToExecute in $scriptsToExecute) {
   $scriptNum++
   Write-Output "----`r`n---- Executing $($scriptToExecute.File) ($($scriptNum) of $($scriptsToExecute.Length))...`r`n----"
   $scriptToExecute | Format-List -Property *
-  & ($scriptToExecute.LocalFilePath) -ScriptProperties $scriptToExecute -SubscriptionId $SubscriptionId -LogicalServerName $LogicalServerName
```

Comment:
Oh, removing this was intentional? :)

I think it may be useful for things like dev vs. prod execution branching. Just one example: When we run the scripts for validation or debugging, we could have verbose tracing enabled by default. When customers run the same script, they could see simplified output. The difference in behavior could be driven by the use of a dev manifest that includes a different set of inputs.

Reply:
Lets sync up, I still don't understand how passing the script we are executing to itself facilitates this. I'd like to see an example of using this.